rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "nightly"
+channel = "stable"
```

Comment:
1. I think you can easily delete this file entirely now.
2. What about rustfmt.toml?

Reply:
1. Removed it.
2. Remove unstable features formatting.